### PR TITLE
fix: esm build compatibility

### PR DESCRIPTION
**Pull request by Sam-Jeston**
# Summary

`getBulkOrderTree` was recently imported from the seaport-js/lib and that broke the esm build fix. As there is no other way to reach this function at the moment, it and depending functions have been temporarily copied over

### CODE REVIEW

**Review by CodeSchwert — DISMISSED** [review]

**Review by wdinau — COMMENTED** [review]
@Sam-Jeston I did a keyword search and found 3 spots with `@opensea/seaport-js/lib/utils` 
one in component, one in transactions and one in tests.
could you please have a look at all three?

File: sdk/package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "jwt-decode": "^3.1.2",
     "lru-memorise": "0.3.0",
     "magic-sdk": "^21.2.0",
+    "merkletreejs": "^0.3.11",
```
Comment (zaidarain1) [inline sdk/package.json:48]: Was wondering if the dependency needs to be added to the SDK package.json as well, if it's already listed in the orderbook package.json dependencies. Would the orderbook not already provide the dependency code when building, similar to other dependencies for other packages?

**Review by zaidarain1 — CHANGES_REQUESTED** [review]

File: sdk/package.json (hunk 1 above)
Comment (Sam-Jeston) [reply sdk/package.json:48]: This is a consequence of `yarn build`, I did not add it manually - any dependency in that package.json's of the packages gets lifted here

Comment (zaidarain1) [reply sdk/package.json:48]: I see, thank you for the clarification. Apologies, was not aware of this 😄

**Review by zaidarain1 — APPROVED** [review]

**Review by naveen-imtb — APPROVED** [review]

**Pull request merged**